Makefile

```diff
@@ -13,7 +13,7 @@ PKGS = $(shell go list ./... | grep -v vendor)
 PKGFILES = $(shell find . \( -path ./vendor -o -path ./Godeps \) -prune \
 		-o -type f -name '*.go' -print)
 PKGFILES_notest = $(shell echo $(PKGFILES) | tr ' ' '\n' | grep -v '\(client/test\|_test.go\)' )
-GOCYCLO ?= 15
+GOCYCLO ?= 20
```

Comment:
Why only on 3.0.x?

Reply:
Tbh, I just assumed we'd bumped it and `3.0` was not, but I see I was wrong, I'll have another look, it's a little odd

Reply:
Yes, absolutely.

For master I smell a QA ticket. 🐕

Reply:
It seems `golangci-lint` does cyclomatic complexity checking ,but it's set at `20`.

This means that the old `Makefile` GoCyclo` check is not really relelevant anylonger.

I'd vote for removing it. @lluiscampos what do you think?

Reply:
Discussed f2f:
- Bumping to 20 for now. @lluiscampos follows up with a ticket for `master` to fix this for `golangci-lint`